### PR TITLE
temporary "fix" for the search bar issue

### PR DESCRIPTION
**Pull request by nubwett**
added a very inefficient way to solve the issue by using a time interval to check if the database is loaded or not

### CODE REVIEW

File: src/pages/BookPage.tsx [hunk 1]
```diff
@@ -14,11 +14,12 @@ import { DbManager } from "../database/DbManager";
  * Use 'useState' for dynamic variables.
  * When the variable changes, the places where it's being used are automatically re-rendered.
  */
+
 const BookPage: React.FC = () => {
   // the search string inputted by the user
   const [searchString, setSearchString] = useState<string>("");
   const [songs, setSongs] = useState<DbSong[]>([]);
-
+  // const [showSearchBar, setShowSearchBar] = useState<boolean>(DbManager.isInitialized());
```
Comment (brotatotes) [inline src/pages/BookPage.tsx:22]: remove this and other commented lines of code

File: src/pages/BookPage.tsx [hunk 2]
```diff
@@ -42,11 +43,24 @@ const BookPage: React.FC = () => {
...
 
+  function checkDatabaseLoad() {
+    if (DbManager.isInitialized()) {
+      // eslint-disable-next-line
+      document.getElementById("searchBar")!.hidden = false;
+      clearInterval(check);
+    }
+  }
+
+  const check = setInterval(checkDatabaseLoad, 1000);
```
Comment (brotatotes) [inline src/pages/BookPage.tsx:54]: This seems very hacky. Is there another way?

**Review by brotatotes — CHANGES_REQUESTED** [review]

**Review by brandonxia01 — COMMENTED** [review]
Thanks for the PR! 

Maybe you can give this a try? - The DB finishes initialization on this line https://github.com/Church-Life-Apps/Songs/blob/master/src/database/DbManager.tsx#L57, so if we can trigger a re-render of the search bar or set `hidden=false` when that code block runs then we will only need to do it once!

**Comment by brotatotes** [comment]
Opening new PR #93

**Pull request closed without merging**